apps/DemoApp/DemoAppUITests/DemoAppUITests.m

```diff
@@ -38,4 +38,32 @@ - (void)testPlayVideo {
     }
 }
 
+- (void)testIMASDK {
+    XCUIApplication *app = [[XCUIApplication alloc] init];
+    [app setLaunchEnvironment:@{@"ENV_KEY": @"tr4q3qahs0gflm8b1c75h49ln", @"TEST_SCENARIO": @"IMA"}];
+    [app launch];
+    XCTestExpectation *exp = [[XCTestExpectation alloc] initWithDescription:@"Wait for launch (~5 sec) and preroll (10 sec)"];
+    XCTWaiterResult result = [XCTWaiter waitForExpectations:@[exp] timeout:15.0];
```

Comment:
is this test reliable? or is there some flakiness due to variation in startup time and/or ad serving behavior?

Reply:
There is some variability that could happen here, but I've tried to choose times that are much longer than expected. If we see a lot of flakiness with this test, it may be worth looking into how to make the ad serving more reliable. But so far in my own testing, I haven't encountered flakiness